README.md

```diff
@@ -284,7 +284,7 @@ client's CA location with `'ssl.ca.location': certifi.where()`.
 Prerequisites
 =============
 
- * Python >= 2.7 or Python 3.x
+ * Python >= 2.7, >=3.9
```

Comment:
If Py 3.9 is broken then this should be <3.9, but we'll want to fix it instead.

Reply:
Whoops. Typo.

Agreed about fixing it instead, but I think for the time being a note that it is broken with 3.9 would be save people a lot of heartache